src/data/faq_de.json

```diff
@@ -87,6 +87,126 @@
                     }
                 ]
             },
+            {
+                "title": "Schnelltest",
+                "id": "rapid_antigen_test",
+                "active": true,
+                "accordion": [
+                    {
+                        "title": "Warum werden jetzt Schnelltests integriert?",
+                        "anchor": "rat_why_integrate",
+                        "active": true,
+                        "textblock": [
+                            "In der Version 2.1 der Corona-Warn-App hat das Projektteam aus Deutscher Telekom und SAP wie angekündigt Schnelltests integriert, sodass Infektionsketten noch schneller unterbrochen werden können.",
+                            "Während PCR-Testergebnisse erst einen Tag später vorliegen und geteilt werden können, können Nutzer*innen positive Schnelltest-Ergebnisse bereits kurz nach dem Test teilen. So werden Kontaktpersonen schneller gewarnt und Infektionskette schneller unterbrochen.",
+                            "Ein negatives Testergebnis kann Nutzer*innen, falls gesetzlich festgelegt, außerdem als Nachweis für das Vorliegen eines negativen Schnelltest-Ergebnisses dienen. Die Anerkennung von Test-Nachweisen kann allerdings von Bundesland zu Bundesland variieren. Nutzer*innen sollten sich deshalb über die jeweiligen Kriterien in ihrem Bundesland informieren."
+                        ]
+                    },
+                    {
+                        "title": "Wie erhalte ich mein Schnelltest-Ergebnis?",
+                        "anchor": "rat_how_get_result",
+                        "active": true,
+                        "textblock": [
+                            "Über den Bereich 'Test registrieren' auf dem Startbildschirm Ihrer Corona-Warn-App, können Sie einen Test hinzufügen, indem Sie den QR-Code scannen. Um das Ergebnis möglichst zeitnah in der App zu erhalten, können Sie den Test in der App aktualisieren, bis das Ergebnis vorliegt. Zwar ruft die Corona-Warn-App Testergebnisse auch automatisch im Hintergrund ab, je nach Betriebssystem kann das aber etwas länger dauern."

```

Comment:
May be "... kann das unterschiedlich lange dauern."

Bei "etwas länger" wäre die Frage, "länger als was?"

Reply:
I think it's clear that länger als means "länger als wenn das Testergebnis manuell aktualisiert wird".

Reply:
thanks for suggest, we keep, but I am open to improve text later!